### PR TITLE
Deploy to paas training

### PR DESCRIPTION
**Pull request by fabio-noumena**

### CODE REVIEW

File: 03_deploying/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# 02 - Modeling a simple use case
```
Comment (jean-noumena) [inline 03_deploying/README.md:1]: That seems off

File: 03_deploying/src/solution/main/npl-v1/.gitkeep
Comment (jean-noumena) [inline 03_deploying/src/solution/main/npl-v1/.gitkeep:1]: what is the purpose of this gitkeep?

File: 03_deploying/README.md (hunk 1 above)
Comment (fabio-noumena) [reply 03_deploying/README.md]: Good catch - forgot about it :D fixed now

File: 03_deploying/src/solution/main/npl-v1/.gitkeep
Comment (fabio-noumena) [reply 03_deploying/src/solution/main/npl-v1/.gitkeep:1]: Let's say "historical reasons"... it's deleted now

**Review by jean-noumena — APPROVED** [review]

File: 03_deploying/src/solution/main/npl-v1/crowdLending_solution03/Loan.npl [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package crowdLending_solution03
+
+@api
```
Comment (jean-noumena) [inline 03_deploying/src/solution/main/npl-v1/crowdLending_solution03/Loan.npl:3]: the @api for the loan is not required

Comment (fabio-noumena) [reply 03_deploying/src/solution/main/npl-v1/crowdLending_solution03/Loan.npl:3]: I would leave it in, in case the reader wants to play with it as well

**Pull request merged**